src/__mocks__/IntersectionObserver.js

```diff
@@ -1,4 +1,7 @@
 const intersectionObservers = []
+const trackedElements = []
+
+module.exports.trackedElements = trackedElements
```

Comment:
I think the naming here is a bit confusing. Maybe `allTrackedElements` or `globallyTrackedElements` or similar to signify that it's a global that affects all instances of `IntersectionObserver`?